tests/contrib/bottle/test_distributed.py

```diff
@@ -30,11 +30,13 @@ def tearDown(self):
         ddtrace.tracer = self._original_tracer
 
     def _trace_app_distributed(self, tracer=None):
+        # DD_BOTTLE_DISTRIBUTED_TRACING enabled by default
         self.app.install(TracePlugin(service=SERVICE, tracer=tracer))
         self.app = webtest.TestApp(self.app)
 
     def _trace_app_not_distributed(self, tracer=None):
-        self.app.install(TracePlugin(service=SERVICE, tracer=tracer, distributed_tracing=False))
+        ddtrace.config.bottle["distributed_tracing"] = False
+        self.app.install(TracePlugin(service=SERVICE, tracer=tracer))
         self.app = webtest.TestApp(self.app)
 
```

Comment:
Just need a test case that uses the DD_BOTTLE_DISTRIBUTED_TRACING environment variable